.github/renovate.json5

```diff
@@ -0,0 +1,134 @@
+// Dependency Update Configuration
+//
+// See https://docs.renovatebot.com/configuration-options/
+// See https://json5.org/ for JSON5 syntax
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base", // https://docs.renovatebot.com/presets-config/#configbase
+    ":semanticCommits", // https://docs.renovatebot.com/presets-default/#semanticcommits
+    ":ignoreUnstable", // https://docs.renovatebot.com/presets-default/#ignoreunstable
+    "group:allNonMajor", // https://docs.renovatebot.com/presets-group/#groupallnonmajor
```

Comment:
This is probably good for one off sample repos or JSS repos but not LANG-docs-samples repos

Reply:
golang-samples is even more aggressive, grouping everything except majors: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/renovate.json

I didn't realize this repo was potentially to springboard new mono-repos. Do you think we should leave this in but add a comment guiding when to remove?

Reply:
We can leave it in. Mono repo support is a secondary goal